src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -44,7 +44,12 @@ public void testFileToFileCRAM(String fileIn, String extOut, String reference) t
                 "-R", refFile.getAbsolutePath()
         };
         runCommandLine(args);
-        SamAssertionUtils.assertSamsEqual(ORIG_BAM, outFile, refFile);
+        //TODO: Make this comparison non-lenient in all cases. Waits on https://github.com/samtools/htsjdk/issues/364
```

Comment:
You need to create a hellbender ticket for this TODO, otherwise we will forget about while waiting for https://github.com/samtools/htsjdk/issues/364 to be fixed.